README.md

```diff
@@ -219,14 +271,14 @@ If you are using a remote database, no changes are needed.
 Set the new image version in an environment variable:
 
 ```shell
-export IMAGE_CONJUR=gcr.io/cloud-marketplace/cyberark/conjur-open-source:1.0
+export NEW_VERSION=mytest
+export IMAGE_CONJUR="gcr.io/cloud-marketplace/cyberark:$NEW_VERSION"
 ```
 
 Update the Deployment definition with the reference to the new image:
 
 ```shell
-kubectl patch deployment $APP_INSTANCE_NAME-conjur \
-  --namespace $NAMESPACE \
+kubectl patch deployment $APP_INSTANCE_NAME \
```

Comment:
I think you want to keep the namespace directive as their kctl may not be set to the right one

Reply:
I'll add a note that the user should set their kubectl context to point to the desired namespace if they haven't already with:
```
kubectl config set-context --current --namespace=<CONJUR-APP-NAMESPACE>
```